Code/Framework/AzToolsFramework/Tests/ComponentModeSwitcherTests.cpp

```diff
@@ -0,0 +1,406 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ComponentModeTestDoubles.h"
+#include "ComponentModeTestFixture.h"
```

Comment:
suggestion: It is probably better to use a full include path here with angle brackets (similar to below) than using a local/relative path. This will mean if we move this file in future it won't suddenly break as the files will be found according to where they actually live